src/types/types.js

```diff
@@ -828,12 +828,15 @@ export type EdgeCurrencyWallet = {
 // swap plugin
 // ---------------------------------------------------------------------
 
+export type EdgeSwapPluginType = 'DEX' | 'CEX' | 'transfer'
+
 /**
  * Static data about a swap plugin.
  */
 export type EdgeSwapInfo = {
   +pluginId: string,
   +displayName: string,
+  +swapPluginType?: EdgeSwapPluginType,
```

Comment:
I think this should be `isDex`, since there could be multiple overlapping attributes going forward: `isTrustless`, `isAtomic`, `isFixedPrice`, or whatever the future brings.

Reply:
That's fair. I can make that change.